app/helpers/requests/application_helper.rb

```diff
@@ -33,33 +33,18 @@ def show_pick_up_service_options(requestable, mfhd_id)
       if requestable.on_shelf?
         display_on_shelf(requestable, mfhd_id)
       else
-        display_requestable_list(requestable.services)
+        display_requestable_list(requestable)
       end
     end
 
     def show_service_options(requestable, _mfhd_id)
-      if requestable.services.empty?
-        content_tag(:div, "#{requestable.title} #{enum_copy_display(requestable.item)} #{I18n.t('requests.no_services.brief_msg')}", class: 'sr-only') +
-          content_tag(:div, I18n.t("requests.no_services.brief_msg"), class: 'service-item', aria: { hidden: true })
+      if requestable.no_services?
+        content_tag(:div, "#{requestable.title} #{enum_copy_display(requestable.item)} #{ActionView::Helpers::SanitizeHelper.sanitize(I18n.t('requests.no_services.brief_msg'))}", class: 'sr-only') +
+          content_tag(:div, ActionView::Helpers::SanitizeHelper.sanitize(I18n.t("requests.no_services.brief_msg")), class: 'service-item', aria: { hidden: true })
       elsif requestable.charged? && !requestable.aeon? && !requestable.ask_me?
         render partial: 'checked_out_options', locals: { requestable: requestable }
       else
-        display_requestable_list(requestable.services)
-      end
-    end
-
-    def show_service_options_fill_in(requestable)
```

Comment:
where was this called? `def show_service_options_fill_in(requestable)`

Reply:
@christinach No where, which is why I removed it.